### PR TITLE
Map Refactoring & Early Polling Locations

### PR DESCRIPTION
**Pull request by lucyq**
map_service contains all map functionality for displaying:
-early polling markers
-directions from home address to polling location
-user precinct

polling_location_finder.js now calls function within map_service to display polling place.

*there may still be some details that need to be worked out.
putting this here so everyone can review and give feedback on refactoring changes

### CODE REVIEW

**Comment by moczka** [comment]
sweet! this will allow us to use the google map instance anywhere within the application. I came across that issue when creating my module. I had to save it globally to the window to have access to it.

**Comment by moczka** [comment]
The part that allows us to get
-directions from home address to polling location 
I'm assuming that's only for election day?

File: app/scripts/map_service.js [hunk 1]
```diff
@@ -0,0 +1,177 @@
...
+  		userInputs.destination = null;
+
+  	}
+
+
+    function createEarlyPollingMarkers() {
+
+        for (var i = 0; i < earlyPollingLocations.length; i++) {
+   
+            var pos = new google.maps.LatLng(earlyPollingLocations[i].position.lat(),
```
Comment (lucyq) [inline app/scripts/map_service.js]: Oh yeah, I think you're right! I'll make these changes later!

**Comment by lucyq** [comment]
Yes, the directions are only for Election Day but the function can be used for early polling locations too

**Comment by lucyq** [comment]
just pulled from master so that my changes are merged with the recent version of master.
also merged tabs.scss and made the change to use the position within the geoJSON file!

**Pull request merged**